storage/pruning/triePruningStorer.go

```diff
@@ -149,7 +149,7 @@ func (ps *triePruningStorer) PutInEpochWithoutCache(key []byte, data []byte, epo
 // GetFromOldEpochsWithoutAddingToCache searches the old epochs for the given key without adding to the cache
 func (ps *triePruningStorer) GetFromOldEpochsWithoutAddingToCache(key []byte) ([]byte, error) {
 	v, ok := ps.cacher.Get(key)
-	if ok {
+	if ok && !bytes.Equal([]byte(common.ActiveDBKey), key) {
```

Comment:
why was this change necessary? It will search for the common.ActiveDBKey in the activePersisters starting with the "oldest" one and returns if it finds one?